cmd/update-main.go

```diff
@@ -274,8 +277,8 @@ func mainUpdate(ctx *cli.Context) {
 		os.Exit(-1)
 	}
 
-	if older != time.Duration(0) {
-		log.Println(colorizeUpdateMessage(downloadURL, older))
+	if updateMsg := computeUpdateMessage(downloadURL, older); updateMsg != "" {
```

Comment:
The key is `older` and the decision needs to be taken based on that ie if `older` is non-zero, there should be `downloadURL`.  `getDownloadURL()` always returns value.